notebooks/02.08-Sorting.ipynb

```diff
@@ -30,7 +30,7 @@
    "cell_type": "markdown",
```

Comment:
Here is another comment for a code cell. This one has execute_result output type. <br/> _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/1/' state='OPEN' cell_index=5>ReviewNB</a>_